src/utils/telemetry/report-error.mjs

```diff
@@ -25,15 +25,19 @@ export const reportError = async function (error, config = {}) {
     return
   }
 
+  // convert a NotifiableError to an error class
+  // eslint-disable-next-line unicorn/no-nested-ternary
+  const err = error instanceof Error ? error : typeof error === 'string' ? new Error(error) : error
```

Comment:
The nested ternary does feel odd to me, but deferring to you since it's not a massively complex statement.

Reply:
with the right prettier settings it would look pretty good IMHO.

```typescript  
const err = error instanceof Error 
  ? error 
  : typeof error === 'string' 
    ? new Error(error) 
    : error
```

whereas with if and assignments it's harder to read IMHO but seems like personal preference